### PR TITLE
markdoc partials

### PR DESCRIPTION
**Pull request by bholmesdev**
#### Description (required)

- Add documentation for Markdoc partials. These allow you to reuse content and reference by relative file path.

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by sarah11918** [comment]
@bholmesdev Just wondering why there is a standalone PR for this, *and* it's added as new in the reorg PR? Can you let me know what your thinking is here?

**Review by sarah11918 — COMMENTED** [review]
Thanks @bholmesdev! This looks super useful! Big comment is in the review suggestion where I showed what this could look like employing some of our ~~tricks~~ strategies!

File: src/content/docs/en/guides/integrations-guide/markdoc.mdx [hunk 1]
```diff
@@ -209,6 +209,30 @@ All Markdown headings will render the `Heading.astro` component and pass the fol
 
 For example, the heading `### Level 3 heading!` will pass `level: 3` and `id: 'level-3-heading'` as component props.
 
+### Partials
```
[the remark this answers is not in the document]
Comment (bholmesdev) [reply src/content/docs/en/guides/integrations-guide/markdoc.mdx:212]: This is fine with me! Hadn't considered we have a partials concept already

File: src/content/docs/en/guides/integrations-guide/markdoc.mdx [hunk 2]
```diff
@@ -209,6 +209,30 @@ All Markdown headings will render the `Heading.astro` component and pass the fol
...
+{% partial file="_footer.mdoc" %}
+```
+
+```md title="src/content/blog/_footer.mdoc"
+Social links:
+
+- [Twitter / X](https://twitter.com/astrodotbuild)
+- [Discord](https://astro.build/chat)
+- [GitHub](https://github.com/withastro/astro)
+```
```
[the remark this answers is not in the document]
Comment (bholmesdev) [reply src/content/docs/en/guides/integrations-guide/markdoc.mdx:234]: Some good suggestions in here! I'll merge and tweak the wording on that one sentence. Thanks 🙌

File: src/content/docs/en/guides/integrations-guide/markdoc.mdx [hunk 3]
```diff
@@ -209,6 +209,34 @@ All Markdown headings will render the `Heading.astro` component and pass the fol
 
 For example, the heading `### Level 3 heading!` will pass `level: 3` and `id: 'level-3-heading'` as component props.
 
+### Markdoc Partials
+
+The  `{% partial %}` tag allows you to render other `.mdoc` files inside your Markdoc content.
```
Comment (sarah11918) [inline src/content/docs/en/guides/integrations-guide/markdoc.mdx:214]:
```suggestion
The `{% partial %}` tag allows you to render other `.mdoc` files inside your Markdoc content.
```

**Pull request merged**